configuration.c

```diff
@@ -695,7 +695,11 @@ const char *config_get_default_audio(void)
       case AUDIO_CTR:
          return "dsp";
       case AUDIO_SWITCH:
+#if defined(HAVE_LIBNX)
+         return "switch_audren";
```

Comment:
As discussed, lets do switch_thread here until audren is sufficient tested by the public, I'd like to have it default by 1.7.8